templates/partials/introduction.html

```diff
@@ -0,0 +1,13 @@
+<div class="container-fluid content-section w-75">
+    <div class="row mt-5">
+        <div class="d-flex flex-column">
+            <h3>{{ title }}</h3>
+            <div>{{ content|safe }}</div>
+        </div>
+         {% with button_type = "med_action_button"%}
```

Comment:
This button looks very different on Figma: there's more padding around the text, the text is vertically centred, the text is smaller, 

Can you please double check the css

Reply:
I think the issue here is the template in the design for a medium button sets the width at 108px but the design for the home page sets this particular button at 126px.  It just happens the text here longer than the text used anywhere else in the design.

Reply:
All of the buttons look different for the reasons I mentioned, not just this one:

>  there's more padding around the text, the text is vertically centred, the text is smaller